src/action/setting.js

```diff
@@ -0,0 +1,22 @@
+import { UNITS } from '../config';
+
+class SettingAction {
+  constructor(store) {
+    this._store = store;
+  }
+
+  //
+  // Bitcoin Unit actions
+  //
+
+  setBitcoinUnit(unit) {
```

Comment:
We should use named parameters like in the other actions. So in this case `({ unit })`.